src/libprojectM/Renderer/Renderer.hpp

```diff
@@ -102,7 +102,7 @@ class Renderer
     bool correction{ true };
 
     bool noSwitch{ false };
-    bool writeNextFrameToFile;
+    bool writeNextFrameToFile{ false };
```

Comment:
what's the purpose of the false? just curious

Reply:
It initializes the member value with `false`, which will prevent an image being dumped on every library init. Being uninialized before, it (mostly) evaluated to a true'ish value (using whatever 32-bit value was present at the memory position on the heap), which caused undefined behaviour - e.g. the image being dumped without triggering.